lib/openxml/pptx/properties/background.rb

```diff
@@ -0,0 +1,23 @@
+module OpenXml
+  module Pptx
+    module Properties
+      class Background < OpenXml::Properties::ComplexProperty
+        include OpenXml::ContainsProperties
+        namespace :p
+        tag :bg
+
+        # ST_BlackWhiteMode - Part 1, 20.1.10.10
+        BLACK_AND_WHITE_MODES = %i{ clr auto gray ltGray invGray grayWhite blackGray
+                                    blackWhite black white hidden }.freeze
```

Comment:
This constant appears in several files. What do you think of extracting it to `OpenXml::Pptx::ST_BlackWhiteMode` like you did in DrawingML?